.github/workflows/pullrequest.yml

```
@@ -223,6 +223,15 @@ jobs:
       - name: Building
         run: ./scripts/ci/run-in-parallel-native.sh build
 
+  docker-build:
+    if: ${{ needs.prepare.outputs.BUILD_CHUNKS && needs.prepare.outputs.DEPLOY_FEATURE == 'true' }}
+    needs:
+      - prepare
+    uses: ./.github/workflows/docker.yml
+    with:
+      projects: ${{ needs.prepare.outputs.BUILD_CHUNKS }}
+      version: ${{ needs.prepare.outputs.DOCKER_TAG }}
```

actionlint (1.7.4)

property "docker_tag" is not defined in object type {build_chunks: string; deploy_feature: string; docker_chunks: string; e2e_build_id: string; e2e_chunks: string; lint_chunks: string; nx_base: string; nx_head: string; test_chunks: string}

(expression)